docs/docs/introduction/getting-started.mdx

```diff
@@ -64,54 +97,64 @@ To start all the default Instant OpenHIE packages, select a deployment platform
 >
 <TabItem value="dockerCompose">
 
-**Before proceeding**, make sure you are in the main `/instant` directory. This directory contains the `instant.sh` script.
+If you have the Instant [OpenHIE codebase](https://github.com/openhie/instant) checked-out make sure you are in the main `/instant` directory.
 
 1. To start up the system, execute the following command:
 
     ```sh
-    ./instant.sh up docker
+    yarn
+    yarn docker:instant init
     ```
 
 1. To stop the running containers, execute the following:
 
     ```sh
-    ./instant.sh down docker
+    yarn docker:instant down
     ```
 
 1. Remove the containers with the command below. However, make sure you **stop all the containers before trying to delete** them.
 
     > **This action will also delete all volumes** created by the containers.
 
     ```sh
-    ./instant.sh destroy docker
+    yarn docker:instant destroy
     ```
 
+Each command also takes a list of package IDs to operate on. If this is left out then all packages are run by default.
+
+If you do not have the code checked-out then you can still startup the environment with the correct command. You can find the docker commands to run in the scripts section of [this file](https://github.com/openhie/instant/blob/ohie-420-docker-outside-of-docker/package.json) in the codebase. Add the action you would like to perform (init, up, down, destroy) to the end of that command.
+
 </TabItem>
 <TabItem value="kubernetes">
 
-**Before proceeding**, make sure you are in the main `/instant` directory. This directory contains the `instant.sh` script.
+For this to work you will need to have Kubernetes enabled in Docker for Desktop or minikube installed and running for Linux.
 
-1. To start up the system, execute the following command:
+If you have the Instant [OpenHIE codebase](https://github.com/openhie/instant) checked-out make sure you are in the main `/instant` directory.
 
-    > This command will output urls from which you can access your Instant OpenHIE instance.
+1. To start up the system, execute the following command:
 
     ```sh
-    ./instant.sh up k8s
+    yarn
```

Comment:
Would we require the user to also run the `build` command before starting the kubernetes instance?

Reply:
It's not required. If you don't build locally it will use the image from docker hub.